client/src/components/friends-page/FriendsContent.js

```diff
@@ -16,6 +15,15 @@ import cloudThree from "../../images/clouds/cloudThree.svg";
 import cloudFour from "../../images/clouds/cloudFour.svg";
 import cloudFive from "../../images/clouds/cloudFive.svg";
 import cloudSix from "../../images/clouds/cloudSix.svg";
+
+import dogOne from "../../images/dogs/dogOne.png";
+import dogTwo from "../../images/dogs/dogTwo.png";
+import dogThree from "../../images/dogs/dogThree.png";
+import dogFour from "../../images/dogs/dogFour.png";
+import dogFive from "../../images/dogs/dogFive.png";
+
+const fakeFrensArray = [dogOne, dogTwo, dogThree, dogFour, dogFive];
```

Comment:
I would recommend to try to create a file, where you just import the images, create the `fakeFrensArray` and export this, so that you just need to import it here.